.tool-versions

```diff
@@ -1,2 +1,3 @@
 terragrunt 0.42.7
 terraform 1.3.7
+python 3.7.16
```

Comment:
Everything looks good, just wondering why this was needed?

Reply:
To ensure Terragrunt uses a compatible python version for some of it's utility scripts. More details in the 
[the commit message](https://github.com/Greater-Wellington-Regional-Council/Environmental-Outcomes-Platform-Infrastructure/pull/66/commits/94bd732f694b65bf475697ba15bf83128405ce8c).